x-pack/legacy/plugins/canvas/canvas_plugin_src/expression_types/embeddable_types.ts

```diff
@@ -6,8 +6,8 @@
 
 // @ts-ignore
 import { MAP_SAVED_OBJECT_TYPE } from '../../../maps/common/constants';
-import { SEARCH_EMBEDDABLE_TYPE } from '../../../../../../src/legacy/core_plugins/kibana/public/discover/embeddable/search_embeddable';
 import { VISUALIZE_EMBEDDABLE_TYPE } from '../../../../../../src/legacy/core_plugins/kibana/public/visualize/embeddable';
+import { SEARCH_EMBEDDABLE_TYPE } from '../../../../../../src/legacy/core_plugins/kibana/public/discover/embeddable/constants';
```

Comment:
dear @elastic/kibana-canvas had to change this import, to ensure, that just this const is imported, and not all of our centralized dependencies of our Discover's NP refactoring

FYI: @flash1293 that's what caused the jest test to fail